src/game/inv.c

```diff
@@ -510,6 +510,17 @@ s32 invGiveWeaponsByProp(struct prop *prop)
 			weapon = prop->weapon;
 			weaponnum = weapon->weaponnum;
 			otherweaponnum;
+#ifndef PLATFORM_N64
+			//This piece of code makes the cheat work in singleplayer. 
+			if (cheatIsActive(CHEAT_DUALWIELDALLGUNS) && !g_Vars.normmplayerisrunning)
```

Comment:
If I understand this line correctly, this prevent the cheat from working in combat simulator, is that right? What's the reason for that? I'm asking cause I'm working on making the "All Guns" cheat work for multiple players, not just solo, and I'm wondering whether it should work just in Co-Op/Counter-Op or also in Combat Simulator.